inst/include/duration.hpp

```diff
@@ -0,0 +1,13 @@
+#ifndef DURATION_HPP
```

Comment:
Should we renamed hpp to h for consistency?

Reply:
Personally, I've always liked `.hpp` as `.h` is ambiguous between C and C++. I'd be more in favor of renaming `date.h` to `date.hpp`...